cpp/server/ct-server.cc

```diff
@@ -222,10 +231,30 @@ int main(int argc, char* argv[]) {
   // For now, run with a dedicated thread pool as the executor for our
   // consistent store to avoid the possibility of DoS through thread starvation
   // via HTTP.
-  ThreadPool etcd_pool(2);
-  EtcdConsistentStore<LoggedCertificate> consistent_store(&etcd_pool,
+  ThreadPool internal_pool(4);
```

Comment:
It's pretty easy to just up this later on when the threads might actually be used, and at least this way we won't have pages of output from `info thread`.
I don't really feel particularly strongly about it though, so feel to try to convince me otherwise if you wish.